BatteryManager.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Globalization;
+using System.Resources;
+namespace BatteryManagementSystem 
+{
+    public class BatteryManager 
+    {
+        IReporter reporter;
+
+        public BatteryManager(IReporter reporterDependency)
+        {
+            if(reporterDependency == null)
+            {
+                throw new InvalidProgramException("Invalid program!!");
+            }
+            reporter = reporterDependency;
+        }
+
+        private static string language = "English";
+        public static string Language
+        {
```

Comment:
Why Battery Manager Class should know the language , ResourceManager existence ....it is required for display or logger

Reply:
Removed Language and ResourceManager existence from BatteryManager class